UI/src/sass/search-result-page-shortlist.scss

```diff
@@ -0,0 +1,25 @@
+.js-enabled .update-my-shortlist-button--apply-visibility {
+  display: none;
+}
+.update-my-shortlist-button--top-margin-ne-2 {
+  margin-top: govuk-spacing(-2);
+}
+
+.my-shortlisted-partners-badge {
+  background-color: govuk-colour("mid-grey");
+  color: govuk-colour("black");
+}
+.my-shortlisted-partners-badge--two-digits-text {
+  background-color: govuk-colour("mid-grey");
+  color: govuk-colour("black");
+  padding: 0.5rem 0.5rem 0.5rem 0.5rem;
+}
+.my-shortlisted-partners-badge--three-digits-text {
+  background-color: govuk-colour("mid-grey");
+  color: govuk-colour("black");
+  padding: 0.7rem 0.4rem 0.4rem 0.4rem;
+}
+//Overriding the default value
+.govuk-checkboxes--small .govuk-checkboxes__label {
```

Comment:
This CSS selector should be more tightly scoped to just the shortlist checkbox, otherwise it might affect other uses of small checkboxes across the site.